### DISCUSSION TITLE
Fluent Editor 2024年规划

### DISCUSSION THREAD

**Discussion by kagol** [opentiny/tiny-editor#20] [General]
如果你对 Fluent Editor 有建议，欢迎评论。

待办任务：
- [x] 增加格式刷功能 @zzxming 
- [x] 支持全屏：页面全屏和沉浸式全屏(`F11`) @zzxming 
- [x] 支持截图直接上传 @zzxming 
- [x] 增加表格行列选择面板，支持插入 `n * n` 表格(目前只能插入 `3*3` 表格) @jany55555 
- [x] 增加基于 Playwright 的 E2E 测试用例 @kagol 
- [x] 完善文档，比如：增加API文档、补充上传图片/文件到服务器的demo、补充插入视频/插入公式的demo等
- [x] 支持插入分割线 @jany55555 
- [ ] 支持斜杆快捷菜单(输入斜线 `/` 出现菜单)
- [ ] 支持气泡菜单(选中内容出现菜单，bubble 主题)
- [ ] 优化超链接的交互
- [ ] 修复图片放大预览功能
- [ ] 优化插入表情，目前插入的表情比较糊
- [ ] 完善每个 demo 的 E2E 测试用例
- [ ] 支持协同编辑
- [ ] 集成AI能力
- [ ] 支持查找替换
- [ ] 支持错别字检查
- [ ] 插入流程图
- [ ] 插入思维导图

---

- [x] 多图片、多文件上传 @qwangry 
- [x] 标题列表 @zzxming 
- [x] 国际化 @zzxming 
- [x] 工具栏提示 @zzxming 
- [x] 可编辑公式 @kiss-keray 
- [x] 生成 TypeScript 类型声明 @kagol 
- [x] 深色模式 @Janson1012

**Comment by zzxming** [comment]
全屏功能是啥意思，F11已经是浏览器的全屏按钮了不是吗？沉浸式全屏是整个页面仅存在编辑器区域吗？

**Reply by kagol** [reply]
> 全屏功能是啥意思，F11已经是浏览器的全屏按钮了不是吗？沉浸式全屏是整个页面仅存在编辑器区域吗？

沉浸式全屏其实就是F11+普通全屏的合集功能
